python/cuml/tsa/holtwinters.pyx

```diff
@@ -484,7 +486,7 @@ class ExponentialSmoothing(Base):
                     raise IndexError("Index input: " + str(index) + " outside "
                                      "of range [0, " + str(self.ts_num) + "]")
                 else:
-                    return cudf.Series(self.level[index])
+                    return cudf.Series(cp.asarray(self.level[index]))
```

Comment:
Unrelated to these change, but still strange overall- while i was looking through this file, I noticed `self.level` is being set to a Python list in the constructor but then being set to a Numba array in `fit()`. At first glance, it seems like this could lead to a confusing error if the model isn't yet trained but code is expecting that to be a Numba array.